srcpkgs/util-linux/template

```diff
@@ -14,7 +14,6 @@ checksum=f261b9d73c35bfeeea04d26941ac47ee1df937bd3b0583e748217c1ea423658a
 
 # Create uuidd system account for uuidd.
 system_accounts="_uuidd"
-_uuidd_homedir="/run/uuidd"
```

Comment:
Maybe set to /var/empty ?

Reply:
This should be /var/empty.

Reply:
it is set to /var/empty as of a day ago, I forgot to ping maxice and resolve